data/etc/foglamp.json

```diff
@@ -0,0 +1,4 @@
+{ "default storage plugin" : "postgres",
+  "storage plugins"        : { "plugin"  : "postgres",
```

Comment:
This should be an array to allow for multiple plugins to be tracked.

"storage plugins" : [ { "plugin" : "postgres", "managed" : true }, { "plugin" : "mysql", "managed" : false } ]